modules/core/src/test/java/org/apache/ignite/internal/processors/cache/consistency/AbstractFullSetCacheConsistencyTest.java

```diff
@@ -0,0 +1,279 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.consistency;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+import javax.cache.CacheException;
+import org.apache.ignite.Ignite;
+import org.apache.ignite.IgniteCache;
+import org.apache.ignite.cache.CacheEntry;
+import org.apache.ignite.internal.util.typedef.G;
+import org.junit.Test;
+
+public abstract class AbstractFullSetCacheConsistencyTest extends AbstractCacheConsistencyTest {
+    /**
+     *
+     */
+    protected static final Consumer<ReadRepairData> GET_CHECK_AND_FIX =
+        (data) -> {
+            IgniteCache<Integer, Integer> cache = data.cache;
+            Set<Integer> keys = data.data.keySet();
+            boolean raw = data.raw;
+            boolean async = data.async;
+
+            assert keys.size() == 1;
+
+            for (Map.Entry<Integer, InconsistentMapping> entry : data.data.entrySet()) { // Once.
+                try {
+                    Integer key = entry.getKey();
+                    Integer latest = entry.getValue().latest;
+
+                    Integer res =
+                        raw ?
+                            async ?
+                                cache.withReadRepair().getEntryAsync(key).get().getValue() :
+                                cache.withReadRepair().getEntry(key).getValue() :
+                            async ?
+                                cache.withReadRepair().getAsync(key).get() :
+                                cache.withReadRepair().get(key);
+
+                    assertEquals(latest, res);
+
+                    checkEvent(data);
+                }
+                catch (CacheException e) {
```

Comment:
Seems, we can remove this catch.
If `CacheException` will be thrown test will fail anyway.
Actually, here you hiding root stack trace.

This, also, applies to other consumers that should be executed normally.